### DISCUSSION TITLE
can we use all python library like opnecv , img2pdf

### DISCUSSION THREAD

**Discussion by saurabhwadekar** [idom-team/idom#986] [Question] [answered]
can we use all python library like opnecv , img2pdf

**Comment by Archmonger** [comment]
Yes. All Python libraries work with ReactPy.

**Reply by rmorshea** [reply, marked as answer]
ReactPy runs server-side and synchronizes JSON serializable "virtual DOM" via a websocket with the client.

**Reply by saurabhwadekar** [reply]
Thanks all
